pkg/apis/extensions/types.go

```diff
@@ -222,6 +222,13 @@ type DeploymentSpec struct {
 	UniqueLabelKey string `json:"uniqueLabelKey,omitempty"`
 }
 
+const (
+	// DefaultDeploymentUniqueLabelKey is the default key of the selector that is added
+	// to existing RCs (and label key that is added to its pods) to prevent the existing RCs
+	// to select new pods (and old pods being select by new RC).
```

Comment:
I suggest adding a "See DeploymentSpec's UniqueLabel field for more information", and also maybe updating the comment there to refer to this field if that doesn't screw up the documentation.